rest-api-spec/src/main/resources/rest-api-spec/test/field_stats/10_basics.yaml

```diff
@@ -59,7 +59,14 @@ setup:
 
 ---
 "Basic field stats":
+  - skip:
+      version: " - 5.99.99"
+      reason:  Deprecation was added in 6.0.0
```

Comment:
is it to not break tests until this is backported to 5.4 like you mention in the PR description?